app/Modules/Front/Presenters/HomepagePresenter.php

```diff
@@ -150,9 +155,11 @@ public function handleFollowTag(string $tagCode): void
         $tagsIds = $this->tagModel->getAll()->where('code', $section->tags)->fetchPairs(null, 'id');
 
         $showAbroad = $this->userModel->showAbroadEvents($this->getUser()->getId());
+
         $this->events = $this->eventModel
             ->getAllWithDates($tagsIds, new DateTime, null, null, $showAbroad);
-        $this->followedTags = $section->tags;
+        // what is this for?
```

Comment:
This is for nothing. Actually, whole `handleFollowTag` and `handleUnfollowTag ` are unfinished work.